### ISSUE TITLE
the number of gt is not matching dt's

### ISSUE THREAD

**Issue by wangyarui** [anibali/eval-mpii-pose#8]
Hi,

Thanks for sharing your code.　I want to evaluate our dataset on single keypoint. but met some issues, I find the number of gt must be match with dt's? Thanks very much.
｀｀｀
uv_error = pos_pred_src - pos_gt_src
ValueError: operands could not be broadcast together with shapes (14,2,1743) (1917,14,2) 
｀｀｀

**Comment by anibali** [comment]
Yes, the number of predictions must match the number of ground truth poses. I'm not sure how it would be possible to compare them if they didn't match.

**Comment by wangyarui** [comment]
@anibali Thanks very much! I will continue to  explore it. I'm pleasure to share it with you if I have some progress.
